apps/trading/.env.capsule

```diff
@@ -18,15 +17,16 @@ NX_MOZILLA_EXTENSION_URL=https://addons.mozilla.org/en-GB/firefox/addon/vega-wal
 NX_ETH_LOCAL_PROVIDER_URL=http://localhost:8545/
 NX_ETH_WALLET_MNEMONIC="ozone access unlock valid olympic save include omit supply green clown session"
 
-# Cosmic elevator flags
+
 NX_SUCCESSOR_MARKETS=false
 NX_STOP_ORDERS=false
 NX_TAKE_PROFIT_STOP_LOSS=true
 NX_ISOLATED_MARGIN=true
-# NX_ICEBERG_ORDERS
-# NX_PRODUCT_PERPETUALS
 NX_METAMASK_SNAPS=false
 NX_REFERRALS=false
+NX_CROSS_CHAIN_DEPOSITS_ENABLED=true
```

Comment:
All env files inherit from the base `.env`, so you do not need to set this here or in any other env file for this app.

Reply:
updated